backend/manager/modules/dal/src/main/resources/bundles/ExecutionMessages.properties

```diff
@@ -226,7 +226,8 @@ job.DetachActionGroupsFromRole=Detaching Action Group ${ActionGroup} from Role $
 job.AddRoleWithActionGroups=Adding Role ${Role} with Action Group ${ActionGroup}
 job.AddSystemPermission=Granting Permission to User/Group ${SubjectName} for System
 job.UpdateStorageDomain=Editing Storage Domain ${Storage} properties
-job.ProcessOvfUpdateForStorageDomain=Initiation OVF_STORE update for storage domain ${Storage}
+job.ProcessOvfUpdateForStorageDomain=Processing OVFs update for Storage Domain ${Storage}
```

Comment:
users are already familiar with the concept of OVF_STORE - why changing that to OVFs?

Reply:
No problem, I assumed that showing user message with underscores in it, is not that user friendly. So changed to what is in the SD menu, i.e., `Update OVFs`.
I reverted the `OVF_STORE` wording.